README.md

```diff
@@ -1,54 +1,76 @@
-* The sketch will warn you that the content of the QSPI flash will be erased. Answer with "Y".
-* When asked if you'd like to use LittleFS on the data partition, select "n". Most of the examples assume that the drive is formatted as FAT. You can use the library to format to LittleFS later. 
-* Reboot the board
- 
-### Formatting the Portenta C33 Internal Storage
-*  Flash the `QSPIFormat` example that can be found in the `Storage` folder for Portenta C33.
-*  Open the Serial Monitor. The sketch will warn you that the content of the QSPI flash will be erased. Answer with "Y".
-*  When asked if you'd like to use LittleFS on the data partition, select "n". Most of the examples assume that the drive is formatted as FAT. You can use the library to format to LittleFS later. 
-*  Reboot the board
+3. Check out the examples or read read the documentation [here](./docs).
 
 ## ✨ Features
+
 * Access files and directories on internal storage, SD cards, and USB mass storage devices.
 * Read and write data to files.
 * Create, remove, and rename files and directories.
 * Copy and move files and directories.
 * List files and subfolders in a directory.
 * Manipulate files and folders from one storage medium to another
 * Format partitions and drives (FAT and LittleFS)
+* Create and modify partitions on the internal storage
+* Register callbacks whenever a USB Mass Storage device has been inserted or removed
+* Get verbose output of every action the library performs for seamless debugging
+
+  
 
 ## Compatibility
+
 This library has been tested with the following STM32 and Renesas based Arduino boards. The availability of storage mediums depends on the hardware interfaces:
-* Portenta Machine Control: USB and Internal QSPI Flash
-* Portenta H7 + Portenta Breakout: USB, SD, and QSPI
-* Portenta H7 + Vision Shield: SD and QSPI
-* Portenta C33 + Portenta Breakout: USB, SD, and QSPI
-* Portenta C33 + Vision Shield: SD and QSPI
-* Opta: Internal QSPI Flash and USB
 
+* **Portenta Machine Control**: USB and Internal QSPI Flash
+* **Portenta H7 + Portenta Breakout**: USB, &sd, and QSPI
+* **Portenta H7 + Vision Shield**: SD and QSPI
+* **Portenta C33 + Portenta Breakout**: USB, &sd, and QSPI
+* **Portenta C33 + Vision Shield**: SD and QSPI
+* **Opta**: Internal QSPI Flash and USB
+
+  
+
+The library allows you to connect a USB drives in a couple of different ways:
 
-Note: Due to an unforeseen compatibility issue on the Portenta Breakout Board, inserting a USB drive on the USB-A port of the breakout board may occasionally cause a reboot on Portenta C33 boards. You can work around this issue by connecting your USB Mass Storage device through a USB hub. This limitation only affects Portenta C33 boards.
+* **Use the USB-C port on the board**: however you have to make sure you are not using the Serial functionality on the same port to avoid conflicts
+* **Connect to the USB-A port** on the breakout board
```

Comment:
Is this limited to the Portenta Breakout with the H7/C33 or can we claim support for the Portenta Machine Control too?

Reply:
we tested with the PMC too, as far as I remember it works fine there, even on the Portenta, it's just on the C33, on the H7 it never happened to me

Reply:
I'd leave the note there tho